relayer/main/main.go

```diff
@@ -275,6 +275,7 @@ func main() {
 			)
 		})
 	}
+	logger.Info("Initialization complete")
```

Comment:
This is not actually safe yet. There is a race condition in the API actually becoming available. We need to add a loop pinging it's own healthcheck before emitting the message. Alternatively we can make the testing code hit the test loop instead of scanning stdout.

I went with the stdout approach to begin with because we were already reading it, and health endpoint isn't yet implemented for the signature aggregator

Reply:
I've integrated the usage of the health endpoints rather than polling stdout, for both the signature-aggregator and the relayer.